indy_node/test/attrib_txn/test_nym_attrib.py

```diff
@@ -0,0 +1,277 @@
+import json
+from contextlib import contextmanager
+
+import base58
+import libnacl.public
+import pytest
+
+from plenum.common.constants import ENC, TXN_ID, TARGET_NYM, \
+    TXN_TYPE, NONCE, STEWARD_STRING
+from plenum.common.exceptions import RequestRejectedException
+from plenum.common.signer_did import DidSigner
+from plenum.common.util import adict, randomString
+from indy_client.client.wallet.attribute import Attribute, LedgerStore
```

Comment:
No code from indy-client can be used anymore

Reply:
this is intermediate PR
the other code will be deleted with next PRs

Reply:
ok